ethereum/core/src/test/java/org/hyperledger/besu/ethereum/privacy/PrivateStateGenesisAllocatorTest.java

```diff
@@ -56,7 +56,7 @@ public class PrivateStateGenesisAllocatorTest {
           List.of(
               new PrivacyGenesisAccount() {
                 @Override
-                public org.hyperledger.besu.plugin.data.Address getAddress() {
+                public org.hyperledger.besu.datatypes.Address getAddress() {
```

Comment:
Might not need to be fully qualified here.

Reply:
fixed